paper/paper.md

```diff
@@ -0,0 +1,201 @@
+
+### Parametric Geometry Tools
+pyGeo contains interfaces to two parametric geometry tools, the Engineering Sketch Pad (ESP) [@Haimes2013a] and OpenVSP [@McDonald2022a].
+ESP is a CAD software, while OpenVSP is a conceptual design tool.
+The two packages have different capabilities, but both directly define the geometry with design variables, and the created geometry can be used in external analysis tools.
+
+pyGeo interfaces with ESP and OpenVSP in similar ways.
+In both cases, pyGeo takes an instance of the model and its points are associated with coordinates in a mesh from a solver in the MACH framework.
+For ESP (\autoref{fig:esp_example}) and OpenVSP models (\autoref{fig:vsp_example}), the pyGeo interface to the respective software stores the model in a form usable within the MACH framework and updates it as design variables are changed throughout the optimization.
+
+![Example of ESP models of hydrogen tanks used through pyGeo [@Brelje2021a]. \label{fig:esp_example}](esp_example.png)
+
+![Example of a VSP aircraft model used through VSP's pyGeo interface [@Yildirim2022a]. \label{fig:vsp_example}](vsp_example.png)
+
+### Class Shape Transformation
+The class shape transformation (CST) methodology [@Kulfan2008] is a popular airfoil parameterization.
+It generates a shape using Bernstein polynomials to scale a class function, which is most often a base airfoil shape.
+pyGeo contains an implementation of this airfoil parameterization that supports design variables for the Bernstein polynomial weightings, the class function parameters, and the airfoil chord length.
+pyGeo's CST implementation can only be used for 2D problems, such as airfoil optimization (\autoref{fig:cst_example}).
+
+![Airfoil defined by three CST coefficients on each surface undergoing a perturbation in one Bernstein polynomial. \label{fig:cst_example}](cst_example.pdf)
+
+## Constraints
+pyGeo also includes geometric constraints to prevent geometrically undesirable designs.
+The most commonly-used class of geometry constraints in pyGeo involves tracking one or more linear dimensions on the optimized object's surface.
+These constraints are created by specifying a single point, a line, or an array of points, along with a normal direction, then computing two line-surface intersection points.
+Some commonly used geometric constraints in shape optimization, such as thickness, area, and volume constraints (\autoref{fig:constraint}) can be computed using variations on this approach, which is computationally cheap and robust [@Brelje2020a].
+
+![Thickness and volume constraints demonstrated on an wing section [@Brelje2020a]. \label{fig:constraint}](constraints_3d.pdf)
+
+If a more complex geometry needs to be integrated into an optimized surface, pyGeo supports an alternative geometric constraint formulation based on arbitrary triangulated surfaces as described in [@Brelje2020a] and illustrated in \autoref{fig:trisurf}.
```

Comment:
I would change this to:
"... arbitrary triangulated surfaces [@Brelje2020a], illustrated in \autoref{fig:trisurf}."